dcp-prototype/frontend/browser/openapi.yml

```diff
@@ -0,0 +1,106 @@
+openapi: 3.0.0
+info:
+  description: |
+    DCP 2.0 Consumer backend API
+  version: "1.0.0"
+  title: DCP 2.0 Consumer 
+servers:
+# Added by API Auto Mocking Plugin
+  - description: SwaggerHub API Auto Mocking
+    url: 'https://api.browser.humancellatlas.org/v1'
+tags:
+  - name: project
+    description: API used to access project information
+  - name: download
+    description: API used to download data
+paths:
+  /projects:
+    get:
+      tags:
+        - project
+      summary: Retrieve filterable information on all projects
+      operationId: getProjects
+      responses:
+        '200':
+          description: OK
+  /projects/{projectUUID}:
+    get:
+      tags:
+        - project
+      summary: |
+        Retrieve detailed information about a project. This includes a list of 
+        downloadable files.
+      operationId: getProjectsUuid
+      parameters: 
+        - name: projectUUID
+          in: path
+          description: The uuid of the project
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Bad Request
+        '404':
+          description: File Not Found
+  /projects/{projectUUID}/files/{fileUUID}:
+    post:
+      tags:
+        - project
+        - download
+      summary: Request to download a file
+      operationId: getProjectFilesUuid
+      parameters:
+        - name: projectUUID
+          in: path
+          description: The uuid of the project
+          required: true
+          schema:
+            type: string
+        - name: fileUUID
+          in: path
+          description: The uuid of the file
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Return a download URL. 
+        '400':
+          description: Bad Request
+        '404':
+          description: File Not Found
+  /files:
```

Comment:
Looking ahead to the 2/21 demo, should we exclude this endpoint until we implement multi-file/packaged downloads?

Reply:
yes, in I'll remove it

Reply:
Also it seems there's no way this could be a synchronous call, right?

Reply:
correct it would not be sync. This was the whole zipping downloads feature that isn't happening.